standalone/alertAndPluginDetails.js

```diff
@@ -0,0 +1,40 @@
+extAlert = org.parosproxy.paros.control.Control.getSingleton().
+    getExtensionLoader().getExtension(
+        org.zaproxy.zap.extension.alert.ExtensionAlert.NAME) 
+
+extAscan = org.parosproxy.paros.control.Control.getSingleton().
+    getExtensionLoader().getExtension(
+        org.zaproxy.zap.extension.ascan.ExtensionActiveScan.NAME);
+
+extPscan = org.parosproxy.paros.control.Control.getSingleton().
+    getExtensionLoader().getExtension(
+        org.zaproxy.zap.extension.pscan.ExtensionPassiveScan.NAME);
+
+var pf = Java.type("org.parosproxy.paros.core.scanner.PluginFactory");
+
+printHeaders();
+
+if (extAlert != null) {

```

Comment:
` && extPscan != null`? or later in the passive check?

Reply:
Added later as suggested.